core/src/components/cat-input/cat-input.tsx

```diff
@@ -325,7 +325,7 @@ export class CatInput {
                 onFocus={this.onFocus.bind(this)}
                 onBlur={this.onBlur.bind(this)}
                 aria-invalid={this.invalid ? 'true' : undefined}
-                aria-describedby={this.hint?.length ? this.id + '-hint' : undefined}
+                aria-describedby={this.hint?.length || this.invalid ? this.id + '-hint' : undefined}
```

Comment:
I think you should use the same logic that is used to show the form hint (L353)

Reply:
done